### ISSUE TITLE
Functionality for Specifying a Generation

### ISSUE THREAD

**Issue by HZ2412** [amharfm/percymon-gen7#1]
Is there any documentation on how to specify a specific generation?

Right now, it is automatically choosing Gen 8 Random Battle Unrated. But Gen 7 and Gen 8 Pokemon are not in the database.

I am wondering if there is a way to specify parameters, or if I can add those data manually? If I can add the data manually, are there other things I need to take a note of, such as modifying elsewhere in the program to let it pick up the newly added Pokemon?

Thanks.

**Comment by amharfm** [comment]
You're right, this runs when only the hype of Gen7 AG happened (no newer generation than gen 7 AG).
I myself have never run this on the current updated Showdown, sorry. Will try to run this AI again once I have some time.

Cheers

**Comment by koudai8** [comment]
I noticed that the typing seems to be a little off when I try to use the bot.

I put "console.log("targetTyping[i]: "+targetTyping[i]);" in tools.js between line 194-195, and got the following output against an opposing Talonflame with Swampert (earthquake/ice ball/superpower).

![image](https://user-images.githubusercontent.com/24820371/113521545-254ef100-9568-11eb-824e-70a68e1319df.png)

where the type of Talonflame seems to be "psychic" and "steel".

And notice that earthquake will be chosen, which isn't an optimal move.

This is quite unexpected; am I doing something wrong here?

Thanks.
